docs/release-notes.md

```diff
@@ -9,6 +9,18 @@ meta:
 ---
 # Release Notes
 
+## 8.2.0
+### New Insights Agent 2.0
```

Comment:
@rbren I copied this off the feature doc pointed to me by Joe. I'm guessing we want to have some sort of link that goes to the changes, especially breaking changes?

Reply:
:nod: let's link here: https://github.com/FairwindsOps/charts/tree/master/stable/insights-agent#version-20